homeassistant/components/nasweb/strings.json

```diff
@@ -0,0 +1,47 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Add NASweb device",
+        "description": "{nasweb_schema_img}NASweb combines the functions of a control panel and the ability to manage building automation. The device monitors the flow of information from sensors and programmable switches and stores settings, definitions and configured actions.",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "missing_internal_url": "Make sure Home Assistant has valid internal url",
+      "missing_nasweb_data": "Something isn't right with device internal configuration. Try restarting the device and HomeAssistant.",
+      "missing_status": "Did not received any status updates within the expected time window. Make sure the Home Assistant Internal URL is reachable from the NASweb device.",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "issues": {
+    "invalid_authentication": {
```

Comment:
I discussed the solution to raise issues with @joostlek a bit, he suggests instead of raising issues the integration should raise translated `ConfigEntryError` exceptions.

I think he's right, and sorry to waste your time.

Raising a `ConfigEntryError` will look like this:

![image](https://github.com/user-attachments/assets/fa48df1e-d7da-45ad-addf-70889c84cd39)

In the screenshot, this is what the integration raised:
```
raise ConfigEntryError(translation_key="config_entry_error", translation_placeholders={"name":"blabla"})
```

And this is what's in `strings.json`:
```json
{
  "exceptions": {
    "config_entry_error_test": {
      "message": "Failed to save item {name} to Bring! list"
    }
  }
}
```

Please change to raise ConfigEntryError instead of creating issues, and I'm sorry for wasting your time.